src/main/java/uk/gov/hmcts/probate/service/probateman/mapper/AliasNameMapper.java

```diff
@@ -7,6 +7,7 @@
 import uk.gov.hmcts.reform.probate.model.cases.CollectionMember;
```

Comment:
It doesn't look like we are using this mapper at all
If so, we should remove it and its Unit Test

Reply:
done